Bio/motifs/__init__.py

```diff
@@ -160,8 +160,12 @@ class Instances(list):
 
     def __init__(self, instances=None, alphabet=None):
         """Initialize the class."""
```

Comment:
The docstring ought to explain the arguments, especially as you are changing how the alphabet argument is interpreted.